resources/public/api/conf/1.0/schemas/create-self-assessment-response.json

```diff
@@ -45,6 +45,12 @@
             "type": "number",
             "description": "The income obtained from employments",
             "example": "13567.77"
+          },
+          "selfEmploymentProfit": {
+            "id": "/properties/saReturns/items/selfEmploymentProfit",
+            "type": "number",
+            "description": "The income obtained for the self employment",
```

Comment:
the profit

Reply:
That's what I had at first, but it doesn't make sense to me. Profit means something completely different.

Reply:
This is what we are returning though. We may want to add later on the income and expenses.

Reply:
done